### PR TITLE
docs(useThrottle): update useThrottle example and sandbox

### PR DESCRIPTION
**Pull request by qiweiii**
#501

### CODE REVIEW

File: docs/useThrottle.md [hunk 1]
```diff
@@ -20,48 +20,72 @@ import { useThrottle } from 'rooks';
...
-      <button onClick={addNumber}>Add number</button>
-      <button onClick={addNumberThrottled}>Add number throttled</button>
-    </>
+    <div>
+      <h1>Rooks : useThrottle example</h1>
+      <input
+        ref={ref}
+        onChange={(e) => {
+          setText(e.target.value);
+          if (isReady) throttledFn();
```
Comment (imbhargav5) [inline docs/useThrottle.md:45]: `isReady` is just an indication for the user. But the conditional is not needed here.

File: docs/useThrottle.md [hunk 2]
```diff
@@ -20,48 +20,72 @@ import { useThrottle } from 'rooks';
...
+function Demo() {
+  const [text, setText] = useState("");
+  const [throttleValue, setThrottleValue] = useState("");
+  const [throttledFn, isReady] = useThrottle(changeThrottleValue, 1000);
+  // isReady is a boolean that tells you whether calling throttledFn at that point
   // will fire or not.
   // Once the timeout of 1000ms finishes, isReady will become true to indicate that the next time
-  // addNumberThrottled is called it will run right away.
+  // throttledFn is called it will run right away.
+  const ref = useRef(null);
```
Comment (imbhargav5) [inline docs/useThrottle.md:32]: What does this `ref` do? 🤔

**Review by imbhargav5 — CHANGES_REQUESTED** [review]

File: docs/useThrottle.md (hunk 1 above)
Comment (qiweiii) [reply docs/useThrottle.md:45]: Got it

File: docs/useThrottle.md (hunk 2 above)
Comment (qiweiii) [reply docs/useThrottle.md:32]: It was from previous code, I think we don't need to use a ref here, I will remove it

**Review by imbhargav5 — APPROVED** [review]

**Review by imbhargav5 — APPROVED** [review]

**Pull request merged**